internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/controllerapi/TestControllerClient.java

```diff
@@ -97,8 +97,8 @@ public void testControllerClientWithInvalidUrls() throws IOException {
     String errorResponseStoreName = Utils.getUniqueString("test-error-store");
     String fakeLeaderControllerUrl = "http://fake.leader.controller.url";
 
-    String nonExistentControllerUrl1 = "http://localhost:22";
-    String nonExistentControllerUrl2 = "http://localhost:23";
+    String nonExistentControllerUrl1 = "http://localhost:1100";
+    String nonExistentControllerUrl2 = "http://localhost:1101";
```

Comment:
Must it be a free port? That could make the test flaky if some other test runs at the same time on the host, right...?

Reply:
Yes. I assumed that using a free port would automatically make a service listen on it. Tested locally that that was wrong. Updating the PR

Reply:
Ok... Using `Utils::getFreePort` also has a chance to be flaky BTW, but I guess we could let the test-retry framework take care of that...

On that note... `Utils::getFreePort` doesn't belong in `venice-common`, IMHO. We should probably move it to `TestUtils` in `venice-test-common`...

Reply:
Oh yeah. For sure. I'll move it to `TestUtils`